POS/Troubleshooting/01_OutOfStockOrder.md

```diff
@@ -1,6 +1,7 @@
 [!!User interface Omni-Channel Orders](../../Channels/UserInterface/05a_Orders.md)
 
 [comment]: <> (add link to Warehouse module if available)
+<!---Error-->
 
```

Comment:
- *Quick Booking* heißt jetzt *Quick posting* in Warehousing. Generell: book -> post, booking -> posting (wie in Buchhaltung). Bitte durchgängig suchen und vereinheitlichen.
- Tab Warehouse/Suppliers -> Tab Warehouse/suppliers (kleines S nach dem Schrägstrich und kein Leerzeichen zwischen Wort und Schrägstrich)
- [ADD BOOKING] -> [ADD POSTING]